dist/redhat/systemd.inc

```diff
@@ -0,0 +1,19 @@
+%define systemd_daemon_reload \
+if [ -x /usr/bin/systemctl ]; then \
+    /usr/bin/systemctl daemon-reload ||: \
+fi \
+%{nil}
+
+%undefine systemd_post
+%define systemd_post() %{expand:%systemd_daemon_reload}
+
+%undefine systemd_preun
+%define systemd_preun() \
+if [ $1 -eq 0 ] && [ -x /usr/bin/systemctl ]; then \
```

Comment:
This is copied code from rpm, which is LGPL. Better not not copy things and just add the daemon-reload code to %postun in the main source.

Reply:
(i.e. in scylla-jmx.spec)